README.md

```diff
@@ -5,6 +5,18 @@
 | <img width="164" height="164" title="" alt="" src="doc/bitcoin.png" /> |
 | [1Cw9nZu9ygknussPofMWCzmSMveusTbQvN](bitcoin:1Cw9nZu9ygknussPofMWCzmSMveusTbQvN) |
 
+## Experimental version of zfs-tools:
```

Comment:
We need to maintain support for `zfs-shell` since not everyone is ready to migrate to ZFS permissions, and people deploy from this repo.

If needed, we can add to `zfs-shell` some sort of canary command that the client can then query, and if that fails, revert back to using `zfs-shell` instead of ZFS permissions.

Reply:
First of all, sorry, the last weeks I was too busy with other things . 

I wonder how we could make zfs-shell optional.

Case "zfs-shell": 
zfs-shell should be installed and registered as a shell. We might even need the sudo rules.
ztools should invoke zfs-shell as a helper script.

Case "zfs-permissions":
zfs-shell should not be installed (or at least) not registered. Sudo rules should not be installed or at least not activated.
ztools should invoke zpool and zfs directly.

Reply:
What about creating two Debian packages out of a single source package, e.g. "zfs-tools" and optional "zfs-shell" (providing zfs-shell and sudo rules). zfs-tools could suggest or even recommend zfs-shell.

Reply:
[Attempt to split the binary packages](https://github.com/MichaelHierweck/zfs-tools/tree/split-into-zfs-tools-and-zfs-shell)